docs/source/getting_started.rst

```diff
@@ -0,0 +1,62 @@
+.. _getting_started:
+
+Getting Started
+===============
+
+I you are a Geoscience ANALYST user, you can skip this section since the installation will be bundled into
+your software and using las-geoh5 is as simple as selecting from menu.  Otherwise, read on to learn how to
+install and use via ui.json file inside or outside of ANALYST.
```

Comment:
I like the idea. but it is sort of assumed that GA users won't come here. We can focus on the "external" people.

Reply:
removed